tests/utils/test_serve_logs.py

```diff
@@ -22,7 +22,9 @@
 import jwt
 import pytest
 import time_machine
+from py import path
```

Comment:
Please use `tmp_path` and `pathlib` instead

Reply:
Thanks. I've used `pathlib` instead.